src/common/handler.ts

```diff
@@ -96,8 +99,11 @@ export function getDataHandler(marketplaceAbi: OffchainMarketplaceAbi, marketpla
     const modifiedTrades: Record<string, ModifiedTrade> = {}
     const modifiedIndexes: Record<string, number> = {}
     let contractStatusAction: ContractStatusAction = undefined
+    let notifyTimestamp: bigint = BigInt(0)
 
     for (const block of ctx.blocks) {
+      const timestamp = BigInt(block.header.timestamp)
+      notifyTimestamp = timestamp > notifyTimestamp ? timestamp : notifyTimestamp
```

Comment:
```suggestion
      notifyTimestamp = timestamp
```
would it always be greater? the blocks are iterated in order, it won't go backwards in time so I think the `>` check might be redundant.